app_dart/lib/src/service/scheduler.dart

```diff
@@ -450,7 +454,33 @@ class Scheduler {
             success = true;
           }
         } else {
-          await luciBuildService.rescheduleUsingCheckRunEvent(checkRunEvent);
+          final String sha = checkRunEvent.checkRun!.headSha!;
+          final String checkName = checkRunEvent.checkRun!.name!;
+          final RepositorySlug slug = checkRunEvent.repository!.slug();
+
+          // TODO: Can't access branch from [checkRunEvent.checkRun.checkSuite] because head_branch is not deserialized
```

Comment:
This can be simplified by doing something similar to above

```dart
final List<github.PullRequest> pullRequests = checkRunEvent.checkRun!.pullRequests ?? <github.PullRequest>[];
for (pr in pullRequests) {}
```

Reply:
I'm not sure if I follow which part of the changes this is talking about.

Reply:
Oh, must be about `findTask`. 🙂

Reply:
In the above ci.yaml check:

```
          final List<github.PullRequest> pullRequests = checkRunEvent.checkRun!.pullRequests ?? <github.PullRequest>[];
          for (github.PullRequest pullRequest in pullRequests) {
            await triggerPresubmitTargets(pullRequest: pullRequest);
            success = true;
          }
```